modules/qna/src/backend/storage.ts

```diff
@@ -211,24 +247,28 @@ export default class Storage {
       return hasMatch && !!_.intersection(contexts, filteredContexts).length
     })
 
-    return filteredQuestions.reverse()
+    if (order === 'asc') {
+      return _.orderBy(filteredQuestions, q => q.data.lastModified?.getTime(), [<'asc' | 'desc'>order])
```

Comment:
you could use the date constrictor here instead of creating a date instance in the storage